scrapers/VRHush.yml

```diff
@@ -0,0 +1,29 @@
+name: "VRHush"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - vrhush.com/scenes
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="container main-item"]
+    scene:
+      Title: $info//h1[@class="latest-scene-title"]/text()
+      Date:
+        selector: $info//div[@class="row latest-scene-meta-1"]/div[@class="col-xs-6 text-left"]/text()
+        parseDate: Jan 2, 2006
+      Details:
+        selector: $info//span[@class="full-description"]/text()
+        concat: " "
+      Tags:
+        Name: $info//p[@class="tag-container"]//a[@class="label label-tag"]/text()
```

Comment:
```yaml
Name: $info//p[@class="tag-container"]//a[contains(@class, "label label-tag")]/text()
```
this catches the prime tags (pink) also

Reply:
Good idea.  I've committed this to code as well.